packages/bahmni-services/src/conditionService/__tests__/conditionService.test.ts

```
@@ -13,11 +21,88 @@ import {
+const mockedGetActiveVisit = getActiveVisit as jest.MockedFunction<
+  typeof getActiveVisit
+>;
+const mockedCreateFhirEncounter = createFhirEncounter as jest.MockedFunction<
+  typeof createFhirEncounter
+>;
+
+const mockCreatedEncounter: Encounter = {
+  resourceType: 'Encounter',
+  id: 'created-enc-server-uuid',
+  status: 'in-progress',
+  class: { system: 'http://terminology.hl7.org/CodeSystem/v3-ActCode', code: 'AMB' },
```

ESLint

[error] Replace `·system:·'http://terminology.hl7.org/CodeSystem/v3-ActCode',·code:·'AMB'` with `⏎····system:·'http://terminology.hl7.org/CodeSystem/v3-ActCode',⏎····code:·'AMB',⏎·`

(prettier/prettier)